### PR TITLE
tests(brand-yml): Add tests for brand.yml

### PR DESCRIPTION
**Pull request by karangattu**
This PR adds playwright tests that check for colors and font-families for multiple elements in the brand.yml example app.
This test will run on Chromium, Safari and Firefox for every commit.

### CODE REVIEW

File: tests/playwright/shiny/brand_kitchensink/_brand.yml
Comment (gadenbuie) [inline tests/playwright/shiny/brand_kitchensink/_brand.yml:1]: Can we re-use the example in `examples/brand` instead of copying it into the tests?

Comment (karangattu) [reply tests/playwright/shiny/brand_kitchensink/_brand.yml:1]: We could, I had to add ids to some elements so the controller can use them in the tests. If you are fine with adding those ids in the example app, we can do that.

Comment (gadenbuie) [reply tests/playwright/shiny/brand_kitchensink/_brand.yml:1]: Yeah, it'd be totally reasonable to include those IDs in the example. I scanned through the `app.py` in this PR and couldn't even find where the IDs were added, which is a good sign that adding them isn't going to "break" anything.

Comment (karangattu) [reply tests/playwright/shiny/brand_kitchensink/_brand.yml:1]: I have addressed it. check it now

File: examples/brand/app.py [hunk 1]
```diff
@@ -61,13 +64,15 @@
...
             ui.card(
                 ui.card_header("Button Variants"),
-                ui.input_action_button("btn1", "Default"),
-                ui.input_action_button("btn2", "Primary", class_="btn-primary"),
-                ui.input_action_button("btn3", "Secondary", class_="btn-secondary"),
-                ui.input_action_button("btn4", "Info", class_="btn-info"),
-                ui.input_action_button("btn5", "Success", class_="btn-success"),
-                ui.input_action_button("btn6", "Warning", class_="btn-warning"),
-                ui.input_action_button("btn7", "Danger", class_="btn-danger"),
+                ui.input_action_button("default_btn", "Default"),
```
Comment (gadenbuie) [inline examples/brand/app.py:67]: This is very much a minor nit, but would you mind if we used IDs like `btn_*`, `value_box_*`, etc? E.g. `btn_default`, `btn_primary`, `value_box_primary`, etc.

File: pyrightconfig.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "_dev",
     "docs",
     "tests/playwright/deploys/*/app.py",
-    "shiny/templates"
+    "shiny/templates",
+    "tests/playwright/shiny/brand_kitchensink"
```
Comment (gadenbuie) [inline pyrightconfig.json:13]: Do we still need this?

```suggestion
```

**Review by gadenbuie — APPROVED** [review]
Just two small comments, otherwise this looks good to me! Thanks @karangattu

File: pyrightconfig.json (hunk 2 above)
Comment (karangattu) [reply pyrightconfig.json]: You are right. removed it in the commit

File: examples/brand/app.py (hunk 1 above)
Comment (karangattu) [reply examples/brand/app.py]: Sure, made the change it the latest commit.

**Pull request merged**